### PR TITLE
DP-4591 Add "more link" to contact us molecule, use on How-To page

### PR DESCRIPTION
**Pull request by jesconstantine**
This PR adds a variant to `@molecules/contact-us` which builds on the preexisting accordion variant and adds a more link to the accordion content.  It also updates the How To page data .json to use this variant in the main content column and switches to use the pre-existing `@molecules/contact-us` variant with linked title in the right rail (added already by `@templates/content-types/details`.

**JIRA ticket**
https://jira.state.ma.us/browse/DP-4591

### To test:

- [ ] Pull branch down, `cd styleguide`, `gulp`
- [ ] Browse to http://localhost:3000/?p=molecules-contact-us-collapsed-with-more-link and observe accordion functionality; verify more link has been added
- [ ] Browse to http://localhost:3000/?p=templates-details and verify that it's still possible to use the sidebar contact us with no linked title
- [ ] Browse to http://localhost:3000/?p=pages-howto and see the proper variants (described above) being used in How To page

### Screenshots:

**How-To sidebar contact with linked title**

image

**How-To main content accordion contact with "learn more" link**

image

### CODE REVIEW

File: styleguide/source/_patterns/05-pages/howto.json [hunk 1]
```diff
@@ -487,12 +487,16 @@
...
+        "decorativeLink": {
+          "href": "#",
+          "text": "Learn more about this organization",
+          "info": "Learn more about this generic"
+        }
       },{
         "accordion": true,
         "isExpanded": false,
         "subTitle": {
-          "href":"",
```
Comment (gender404) [inline styleguide/source/_patterns/05-pages/howto.json:505]: Why has the `href` variable been removed from subTitle?  It's part of the data model for Column Heading so it should be there, but set to an empty string.

**Comment by gender404** [comment]
merge conflicts exist in this PR, please fix

File: styleguide/source/_patterns/05-pages/howto.json (hunk 1 above)
Comment (jesconstantine) [reply styleguide/source/_patterns/05-pages/howto.json:505]: Fixed in https://github.com/massgov/mayflower/pull/542/commits/5bcd9a3e148d2a7ecbb04ae7e90a8981e0096d9c

**Comment by jesconstantine** [comment]
@legostud I believe I've addressed all of your concerns and @cmajel has 👍  the visual design for the changes.

**Review by gender404 — APPROVED** [review]

**Pull request merged**